examples/em-drift/2fluid.cxx

```diff
@@ -47,10 +47,10 @@ class EMdrift : public PhysicsModel {
   FieldGroup comms;
 
   // Inverts a Laplacian to get potential
-  Laplacian *phiSolver;
+  std::unique_ptr<Laplacian> phiSolver;
```

Comment:
Do these default initialise to `nullptr` like the others?

Reply:
I believe so. I might have fixed this one manually while iterating on the upgrader tool